styles.css

```diff
@@ -515,8 +507,113 @@ a {
   }
 }
 
-@media (min-width: 710px) {
+@media (min-width: 600px) {
```

Comment:
- [OPTIONAL] As the project requirements described, the desktop view starts after 768px wide. This is still a 'mobile view' on its requirements. Try changing it later.
 
<img width="952" alt="image" src="https://user-images.githubusercontent.com/61919617/186473243-429f1b8c-f7fa-4aec-8f5b-a4aa47249c4b.png">